packages/new-polymath-issuer/src/pages/DividendsWizard/Step-2/TaxWithholdingModal.tsx

```diff
@@ -52,34 +54,45 @@ export const TaxWithholdingModal: FC<Props> = ({
 
     const valueAddress = value[csvEthAddressKey].toUpperCase();
 
-    const matchingIndex = findIndex(
-      formTaxWithholdings,
-      taxWithholding =>
-        taxWithholding[csvEthAddressKey].toUpperCase() === valueAddress
-    );
+    // Check if excluded
+    const excluded = exclusionList.find(address => {
+      return address === valueAddress;
+    });
 
-    const alreadyExists = matchingIndex !== -1;
+    if (!excluded) {
+      const matchingIndex = findIndex(
+        formTaxWithholdings,
+        taxWithholding =>
+          taxWithholding[csvEthAddressKey].toUpperCase() === valueAddress
+      );
 
-    if (isEditing || alreadyExists) {
-      const finalValue = { ...value };
+      const alreadyExists = matchingIndex !== -1;
 
-      formTaxWithholdings.splice(matchingIndex, 1, finalValue);
-      form.setFieldValue('taxWithholdings', formTaxWithholdings);
-    } else {
-      const finalValue = { ...value };
+      if (isEditing || alreadyExists) {
+        const finalValue = { ...value };
 
-      form.setFieldValue('taxWithholdings', [
-        ...formTaxWithholdings,
-        finalValue,
-      ]);
-    }
+        formTaxWithholdings.splice(matchingIndex, 1, finalValue);
+        form.setFieldValue('taxWithholdings', formTaxWithholdings);
+      } else {
+        const finalValue = { ...value };
 
-    form.setFieldValue(field.name, {
-      [csvTaxWithholdingKey]: null,
-      [csvEthAddressKey]: '',
-    });
-    form.setFieldTouched(field.name, false);
-    onClose();
+        form.setFieldValue('taxWithholdings', [
+          ...formTaxWithholdings,
+          finalValue,
+        ]);
+      }
+      form.setFieldValue(field.name, {
+        [csvTaxWithholdingKey]: null,
+        [csvEthAddressKey]: '',
+      });
+      form.setFieldTouched(field.name, false);
+      onClose();
+    } else {
+      form.setFieldError(
+        'currentTaxWithholding.Investor ETH Address',
+        'Address excluded'
```

Comment:
I'd change this error message to something more explicit like `This address was excluded in the previous step`